schedule/yast/btrfs/btrfs+warnings_opensuse.yaml

```diff
@@ -49,17 +49,13 @@ test_data:
       partitions:
         <<: !include test_data/yast/btrfs/common/btrfs+warnings_partitions.yaml
         rootfs_small:
-          - role: raw-volume
+          - role: raw
             size: 2mb
-            id: bios-boot
-          - role: operating-system
+            id: BIOS Boot Partition
```

Comment:
This is also a point to discuss, as for role we map values, but for `id` we use test_data. This is inconsistent. Also this solution would require changes in all yamls in case there is a rename. Mapping allows to have a change in a single place, which is beneficial for our setup, as we support multiple versions of distris. For the same scenario we would have to double number of yamls with the test data. After all, it has an advantage of keeping the implementation free of magic strings.